src/NzbDrone.Core/MediaFiles/EpisodeImport/Aggregation/Aggregators/Augmenters/Language/AugmentLanguageFromFileName.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using NzbDrone.Core.Download;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.MediaFiles.EpisodeImport.Aggregation.Aggregators.Augmenters.Language
+{
+    public class AugmentLanguageFromFileName : IAugmentLanguage
+    {
+        public int Order => 1;
+        public string Name => "FileName";
+
+        public AugmentLanguageResult AugmentLanguage(LocalEpisode localMovie, DownloadClientItem downloadClientItem)
+        {
+            var languages = new List<Languages.Language> { localMovie.FileEpisodeInfo?.Language };
+
+            if (languages == null)
```

Comment:
If `localMovie.FileEpisodeInfo?.Language` is null this will create an array with a null value, instead:

```
if (localMovie.FileEpisodeInfo?.Language == null)
{
  return null;
}

return new AugmentLanguageResult(localMovie.FileEpisodeInfo?.Language, Confidence.Filename);
```

Is better.